specs/Accessibility.md

```diff
@@ -0,0 +1,72 @@
+# Requirements
+
+## ARIA conformance
+In general, components need to conform to the [WAI-ARIA Authoring Practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/). These practices are evaluated to achieve best possible user experience with supported browser/screen reader combinations.
+
+## Keyboard navigation
+- All elements that are interactive need to be focusable using keyboard keys (TAB/Shift+Tab or arrow keys)
+- Focused element needs to be highlighted if it was focused by keyboard
+- Focus should never go to the ``<body>`` element
+- User needs to be able to move focus at any time, focus should not be be 'stuck' on one element
+- In RTL mode, left arrow key moves the focus to the next focusable element, right arrow key moves the focus to the previous focusable element within the focus zone.
+
+## Focus
+- Whenever possible, focus should land only on elements that have concrete implicit or explicit ARIA role (`<button />`, `role='button'`, `role='menuitem'`, ....)
+- onClick event handlers need to be handled on focusable elements with implicit or explicit ARIA role and not on their parent/child elements
+- When reacting on focus or on hover screen reader interaction needs to be specified. Some screen readers do not have concept of hovering and not always focus elements, so there needs to be another way for the screen reader users to interact with such elements. For example showing popups on focus or on hover needs to be justified and alternative interaction compatible with the screen readers need to be provided
+
+## Screen reader support
+User needs to be able to interact with the application using:
+- screen reader with virtual cursor navigation
```

Comment:
I propose to add definition of these modes

- screen reader with virtual cursor navigation (Browse Mode)
- screen reader without virtual cursor navigation (Application Mode)

Good application mode is useful for power users too